kubernetes_platform/python/README.md

```diff
@@ -102,3 +102,29 @@ def my_pipeline():
     delete_pvc1 = kubernetes.DeletePVC(
         pvc_name=pvc1.outputs['name']).after(task2)
 ```
+
+### Pod Metadata: Add pod labels and annotations to the container pod's definition
+```python
+from kfp import dsl
+from kfp import kubernetes
+
+
+@dsl.component
+def comp():
+    pass
```

Comment:
I don't think is a good idea to call Kubernetes API inside the pod because KFP Tasks don't have permission to any resources outside of the pod by default. We can extend this example to use Kubernetes downstream API definition when we introduce the Kubernetes downstream proto spec to KFP in the future.

Reply:
There will be no conflict in imports if we don't need to call the python-kubernetes client.

Reply:
OK.